Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2437,6 +2437,69 @@ public async Task HandleReorgInSynchronizing()
             }
         }
 
+        [Fact(Timeout = Timeout)]
+        public async void RestageTransactionsOnceLocallyMinedAfterReorg()
+        {
+            var policy = new BlockPolicy<DumbAction>(new MinerReward(1));
+            var minerA = CreateSwarm(TestUtils.MakeBlockChain(policy, new DefaultStore(null)));
+            var minerB = CreateSwarm(TestUtils.MakeBlockChain(policy, new DefaultStore(null)));
+
+            var privateKeyA = new PrivateKey();
+            var privateKeyB = new PrivateKey();
+
+            try
+            {
+                await StartAsync(minerA);
+                await StartAsync(minerB);
+
+                const string dumbItem = "item0.0";
+                var txA = minerA.BlockChain.MakeTransaction(
+                    privateKeyA,
+                    new[] { new DumbAction(_fx1.Address1, dumbItem), });
+                var txB = minerB.BlockChain.MakeTransaction(
+                    privateKeyB,
+                    new[] { new DumbAction(_fx1.Address2, dumbItem), });
+
+                Log.Debug("Make minerB's chain longer than minerA's chain.");
+                var blockA = await minerA.BlockChain.MineBlock(minerA.Address);
+                var blockB = await minerB.BlockChain.MineBlock(minerB.Address);
+                var blockC = await minerB.BlockChain.MineBlock(minerB.Address);
+
+                await BootstrapAsync(minerA, minerB.AsPeer);
+
+                Assert.Equal((Text)dumbItem, minerA.BlockChain.GetState(_fx1.Address1));
+                Assert.Equal((Text)dumbItem, minerB.BlockChain.GetState(_fx1.Address2));
+
+                Log.Debug("Reorg occurred.");
+                minerB.BroadcastBlock(blockC);
+                await minerA.BlockAppended.WaitAsync();
```

Comment:
… or it might be better to write the log message at this time.

```suggestion
                await minerA.BlockAppended.WaitAsync();
                Log.Debug("Reorg occurred.");
```